minigalaxy/installer.py

```diff
@@ -292,6 +298,20 @@ def remove_installer(game, installer):
 
 def postinstaller(game):
     err_msg = ""
+    # To prevents crash at launch for Dragon Age : Origins (only when the game is extracted by Innoextract) :
+    # 012c:err:module:import_dll Library PhysXLoader.dll (which is needed by [...]\\NxCharacter.dll") not found
+    # 012c:err:module:import_dll Library NxCharacter.dll (which is needed by [...]\\DAOrigins.exe") not found
+    # 012c:err:module:import_dll Library PhysXLoader.dll (which is needed by [...]\\DAOrigins.exe") not found
+    # 012c:err:module:LdrInitializeThunk Importing dlls for [...]\\DAOrigins.exe" failed, status c0000135
+    # it's needed to install Physx with wine after extraction.
+    postinst_wine = os.path.join(game.install_dir, "redist", "PhysX_9.09.0408_SystemSoftware.exe")
```

Comment:
Before to do the PR, i tried to install Physx only for Dragon Age : Origin. 
In the code, i did in the postinstaller def an IF condition like that : 
```
if game.id == "1949616134":
    postint_wine = ...
    [...]
```
But it did not work and i don't know why :(

And i don't understand why do you want to blacklist applications in redist, these files are not launched automatically.

Reply:
Because game.id is int not str.

Reply:
There will be more games which need Physx, though, and the version in the installer won't be the same every time. That's my point.

Reply:
I thought to use winetricks instead of this solution like this : 
```
def postinstaller(game):
    err_msg = ""
    if game.platform == "windows":
        if shutil.which("winetricks"):
            # Dragon Age Origins needs Physx to launch.
            if str(game.id) == "1949616134":
                prefix_dir = os.path.join(game.install_dir, "prefix")
                command = ["env", "WINEPREFIX={}".format(prefix_dir), "winetricks", "-q", "physx"]
                stdout, stderr, exitcode = _exe_cmd([command])
                if exitcode not in [0]:
                    err_msg = "Postinstallation failed: {}".format(command)
            # Horizon Zero Dawn needs MFC140.dll to launch.
            if str(game.id) == "1949616134":
                prefix_dir = os.path.join(game.install_dir, "prefix")
                command = ["env", "WINEPREFIX={}".format(prefix_dir), "winetricks", "-q", "mfc140"]
                stdout, stderr, exitcode = _exe_cmd([command])
                if exitcode not in [0]:
                    err_msg = "Postinstallation failed: {}".format(command)
    else:
        postinst_script = os.path.join(game.install_dir, "support", "postinst.sh")
        if os.path.isfile(postinst_script):
            os.chmod(postinst_script, 0o775)
            stdout, stderr, exitcode = _exe_cmd([postinst_script])
            if exitcode not in [0]:
                err_msg = "Postinstallation script failed: {}".format(postinst_script)
    return err_msg
```

But it crashes with this message : `TypeError: expected str, bytes or os.PathLike object, not tuple`

Reply:
Using winetricks for this may work, how does that look for the user? Does winetricks show an installer?

Reply:
The users does not see the installer if "-q" command line argument is set.